docs/api/alerts/health.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[alerts-api-health]]
+=== Get Alerting framework health API
+++++
+<titleabbrev>Get Alerting framework health</titleabbrev>
+++++
+
+Retrieve the health status of the Alerting framework.
+
+[[alerts-api-health-request]]
+==== Request
+
+`GET <kibana host>:<port>/api/alerts/_health`
+
+[[alerts-api-health-codes]]
+==== Response code
+
+`200`::
+    Indicates a successful call.
+
+[[alerts-api-health-example]]
+==== Example
+
+Retrieve the Alerting framework health status:
+
+[source,sh]
+--------------------------------------------------
+$ curl -X GET api/alerts/_health
+--------------------------------------------------
+// KIBANA
+
+The API returns the following:
+
+[source,sh]
+--------------------------------------------------
+{
+   "isSufficientlySecure":true,
+   "hasPermanentEncryptionKey":true,
+   "alertingFrameworkHeath":{
+      "decryptionHealth":{
+         "status":"ok",
+         "timestamp":"2021-02-10T23:35:04.949Z"
+      },
+      "executionHealth":{
+         "status":"ok",
+         "timestamp":"2021-02-10T23:35:04.949Z"
+      },
+      "readHealth":{
+         "status":"ok",
+         "timestamp":"2021-02-10T23:35:04.949Z"
+      }
+   }
+}
+--------------------------------------------------
+
+The health API response contains the following properties:
+
+[cols="2*<"]
+|===
+
+| `isSufficientlySecure`
+| Returns`false` if security is enabled, but TLS is not.
+
+| `hasPermanentEncryptionKey`
+| Return the state true if Encryption Saved Object plugin has a permanent encryption Key
```

Comment:
Should this be Returns `true` to be consistent? Also, should it be `Encrypted` instead of `Encryption`?